web/src/beta/features/Editor/DataSourceManager/WmsTiles/index.tsx

```diff
@@ -0,0 +1,120 @@
+import { FC, useState } from "react";
+
+import Button from "@reearth/beta/components/Button";
+import generateRandomString from "@reearth/beta/utils/generate-random-string";
+
+import { DataProps } from "..";
+import {
+  AddLayerWrapper,
+  AssetWrapper,
+  ColJustiftBetween,
+  DeleteLayerIcon,
+  Input,
+  InputGroup,
+  LayerWrapper,
+  SubmitWrapper,
+} from "../utils";
+
+const WmsTiles: FC<DataProps> = ({ sceneId, onSubmit, onClose }) => {
+  const [urlValue, setUrlValue] = useState("");
+  const [layerValue, setLayerValue] = useState("");
+  const [layers, setLayers] = useState<string[]>([]);
+
+  const handleAddLayer = () => {
+    const exist = layers.some((layer: string) => layer === layerValue);
+    if (layerValue.trim() !== "") {
+      if (!exist) setLayers(prev => [...prev, layerValue]);
+      setLayerValue("");
+    }
+  };
+
+  const handleDeleteLayer = (idx: number) => {
+    const updatedLayers = [...layers];
+    updatedLayers.splice(idx, 1);
+    setLayers(updatedLayers);
+  };
+
+  const handleSubmit = () => {
+    onSubmit({
+      layerType: "simple",
+      sceneId,
+      title: generateRandomString(5),
+      visible: true,
+      config: {
```

Comment:
Can you and @pyshx discuss where might a good place be to write and export all format type's config objects? 
Even though I am not sure we need to use them in multiple places, I still feel like having them all in one place would be good 🙏🏻